index.html

```diff
@@ -37,6 +32,119 @@ <h3>I’am  web a developer.</h3>
             <li><img src="images/medium.png" alt="social-icon" /></li>
       </ul>   
     </section>
+       <section class="my-works1">
```

Comment:
- [ ] [Optional] This section tag should wrap all the "works" on your page. I strongly suggest fixing this issue.